packages/devtools-modules/client/shared/components/splitter/Draggable.js

```diff
@@ -19,24 +19,25 @@ const Draggable = React.createClass({
 
   startDragging(ev) {
     ev.preventDefault();
-    const doc = ReactDOM.findDOMNode(this).ownerDocument;
-    doc.addEventListener("mousemove", this.onMove);
-    doc.addEventListener("mouseup", this.onUp);
+    const elt = ReactDOM.findDOMNode(this);
+    elt.setCapture(true);
+    elt.addEventListener("mousemove", this.onMove);
+    elt.addEventListener("mouseup", this.onUp);
     this.props.onStart && this.props.onStart();
   },
 
   onMove(ev) {
     ev.preventDefault();
-    // Use screen coordinates so, moving mouse over iframes
-    // doesn't mangle (relative) coordinates.
-    this.props.onMove(ev.screenX, ev.screenY);
+    // because we use setCapture, the event's properties are correct even with
+    // iframes.
+    this.props.onMove(ev);

```

Comment:
passing the whole event as there are a lot of useful properties there. We might want to create a new object instead but I'd rather not because of GC (mousemove events happen a lot)